index.js

```diff
@@ -6,18 +6,18 @@
 "use strict";
 
 function exportToJsonString(idbDatabase, cb) {
-  let exportObject = {};
+  const exportObject = {};
```

Comment:
This is never re-assigned so can be const. It's a confusing case, because we are updating the object so it seems like it shouldn't be const. But since we are never replacing the object in the variable with a new object, it should.